mlir/test/CMakeLists.txt

```diff
@@ -125,6 +125,10 @@ set(MLIR_TEST_DEPENDS
 if(NOT MLIR_STANDALONE_BUILD)
   list(APPEND MLIR_TEST_DEPENDS FileCheck count not split-file yaml2obj)
 endif()
+# Examples/standalone/test.toy (vis-a-vis the standalone example) depends on these.
+if(LLVM_INCLUDE_EXAMPLES)
+  list(APPEND MLIR_TEST_DEPENDS MLIRCAPIIR MLIRCAPIArith)
```

Comment:
Another question: these aren't already dependencies of the various `mlir-capi-*-test` listed above?